src/components/ebay-combobox/component.ts

```diff
@@ -342,6 +354,8 @@ export default class Combobox extends Marko.Component<Input, State> {
             this._floatingLabel.destroy();
             this._floatingLabel = null;
         }
+
+        // this.dropdownUtil?.cleanup();
```

Comment:
Is this intentional? Should either be removed or uncommented

Reply:
Will remove.